ota-plus-web/app/reactapp/src/utils/WebsocketHandler.js

```diff
@@ -52,7 +52,7 @@ const WebsocketHandler = (function (wsUrl, stores) {
                     }
                     break;
                 case "DeviceSystemInfoChanged":
-                    stores.devicesStore.fetchDeviceNetworkInfo(data.uuid);
+                    stores.devicesStore.fetchDeviceNetworkInfo(data.uuid, {isFromWs: true});
```

Comment:
Check device here.

Reply:
this is further fix to the fetching network info, related to: https://github.com/advancedtelematic/ota-plus-server/commit/042d2b311a7d80dc31d32bf9b702d3c41d4b55b2
file: ota-plus-web/app/reactapp/src/stores/DevicesStore.js, line 223

This code should stay as it is until stores refactoring.